docs/content/en/docs/observability/loki.md

```diff
@@ -39,52 +39,111 @@ For this case, we'll use Helm.
+        source_labels:
+          - __meta_kubernetes_namespace
+        target_label: namespace
+      - action: replace
+        replacement: $1
+        separator: /
+        source_labels:
+          - namespace
+          - app
+        target_label: job
+      - action: replace
+        source_labels:
+          - __meta_kubernetes_pod_name
+        target_label: pod
+      - action: replace
+        source_labels:
+          - __meta_kubernetes_pod_container_name
+        target_label: container
+      - action: replace
+        replacement: /var/log/pods/*$1/*.log
+        separator: /
+        source_labels:
+          - __meta_kubernetes_pod_uid
+          - __meta_kubernetes_pod_container_name
+        target_label: __path__
+      - action: replace
+        replacement: /var/log/pods/*$1/*.log
+        regex: true/(.*)
+        separator: /
+        source_labels:
+          - __meta_kubernetes_pod_annotationpresent_kubernetes_io_config_hash
+          - __meta_kubernetes_pod_annotation_kubernetes_io_config_hash
+          - __meta_kubernetes_pod_container_name
+        target_label: __path__
+      - action: labelmap
+        regex: __meta_kubernetes_pod_label_(.+)
+EOF
 ```
 
-This will install Grafana in the `grafana` namespace, with helm release name as `fission-logs`
+```bash
+$ helm upgrade -n monitoring --install promtail grafana/promtail -f promtail-config.yaml
+```
+
+This will install Prom in the `monitoring` namespace!
+Check if there're is a promtail pod running.
+
+We can access the Promtail UI at `localhost:3101` to see all of the pods logs being tailed along with the labels assigned to them.
+```bash
+$ kubectl --namespace monitoring port-forward $(kubectl  --namespace monitoring get pod -l app.kubernetes.io/instance=promtail -o name) 3101:3101
+```
 
 #### Accessing Grafana UI
 
-The installation above creates a Service in `grafana` namespace.
+If you've followed the above steps, `loki-stack` creates a grafana Service in the `monitoring` namespace.
 To access this, you can use Kubernetes port forwarding.
 
 ```bash
-$ kubectl --namespace grafana port-forward svc/fission-logs-grafana 3000:80
+$ kubectl --namespace monitoring port-forward svc/fission-logs-grafana 3000:80
+```
+
+You'll need to obtain a username and password to access the UI. 
+The username is `admin` and the password can be obtained from inside the kubernetes cluster.
+```bash
+$ kubectl --namespace monitoring get secrets loki-grafana -o go-template --template='{{index .data "admin-password"}}' | base64 -d
 ```
-  
+
 #### Adding Loki as a data source in Grafana
 
 Clicking on the Settings icon in the left pane will bring up a menu, click on `Data Sources`.
 Clicking on `Add Data Source` and select Loki.
-Under HTTP, in the URL field, put the ClusterIP followed by port number.
-You can get the ClusterIP using
-
-```bash
-$ kubectl get svc  -l "app=loki" -o jsonpath="{.items[0].spec.clusterIP}"
-```
+Under HTTP, in the URL field put `http://loki:3100`
```

Comment:
This should be `localhost` ?

Reply:
I don't think localhost will work here. I believe localhost is accessible by containers running in the same pod right?
http://loki:3100 refers to the k8s Service object of Type ClusterIP which is created in the helm chart.